src/WorldCup/ScoreBoard.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Cheba\PhpUnit\WorldCup;
+
+class ScoreBoard
+{
+    protected array $games = [];
+    public function startGame($homeTeam, $awayTeam): object
+    {
+        return new Game($homeTeam, $awayTeam);
+    }
+
+    public function updateScore($game, $homeScore, $awayScore): void
```

Comment:
bugs:
* allow to set as score, any value - like string, or another object?
* allow to set finished game and change score
* if properties of games are public, that method is pointless - developer can change them themselfe

Reply:
Added private properties, setters with validation and checked for a completed game.